chat-widget/src/components/webchatcontainerstateful/common/utils/chatAdapterUtils.ts

```diff
@@ -0,0 +1,79 @@
+import { uuidv4 } from "@microsoft/omnichannel-chat-sdk";
+import { Activity, Attachment, Message, User } from "botframework-directlinejs";
+import { Subscriber } from "rxjs/Subscriber";
+
+export const customerUser: User = {
+    id: "usedId",
+    name: "User",
+    role: "user"
+};
+
+export const botUser: User = {
+    id: "botId",
+    name: "Bot",
+    role: "bot"
+};
+
+// WebChat expects an "echo" activity to confirm the message has been sent successfully
+export const postEchoActivity = (activityObserver: Subscriber<Activity> | undefined, activity: Message, user: User, delay = 1000): void => {

```

Comment:
is A11Y in the picture?, are we testing the reading of the messaes using screen readers?

Reply:
I tried with `Narrator` and reader messages is working. There is not UI changes, we're only manipulating the internals to render sample messages. So, all the messages are rendered as how it would be rendered with a real connection.